lib/drivers/abbott/abbottFreeStyleLibre.js

```diff
@@ -169,7 +172,24 @@ export default function (config) {
       progress(0);
 
       if (data.deviceInfo.productId === FSLIBRE_PRO_PRODUCT_ID) {
-        data.deviceInfo.deviceId = `${data.deviceInfo.driverId}Pro-${data.deviceInfo.serialNumber}`;
+        let generatedSensorId = '';
+        const TABLE_ID_OFFSET = 0;
+        // find oldest historical data packet
```

Comment:
Can we add a comment here about why we use the oldest historical data packet to generate the sensor ID?
Something like "The oldest historical data packet contains a timestamp and the local timezone. Although the hashed value is not globally unique, it is unique for this device, and combined with the device ID is globally unique".

Reply:
Thanks for pointing this out. I added an explanation.